packages/client/src/views/Performance/Statistics.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { IStoreState } from '@client/store'
+import { getOfflineData } from '@client/offline/selectors'
+import { injectIntl } from 'react-intl'
+import { Activity } from '@opencrvs/components/lib/icons'
+import { DashboardEmbedView } from '@client/views/Performance/Dashboard'
+
+const performanceStatisticsComponent = () => (
+  <DashboardEmbedView
+    title={'Statistics'}
+    url={window.config.STATISTICS_DASHBOARD_URL}
+    icon={<Activity />}
+  />
+)
```

Comment:
- think we could export this directly (see previous comments)
- please localize "Statistics"